packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -351,6 +351,22 @@ const ruleset: any = {
       },
     },
 
+    // RPC Code: RPC-Get-V1-14
+    GetMustNotBeLRO: {
+      description: "The Get call cannot be Long Running Operation and it must not have x-ms-long-running-operation property.",
```

Comment:
>The Get call cannot be Long Running Operation and it must not have x-ms-long-running-operation property [](http://example.com/codeflow?start=20&length=103)

The GET operation call cannot be Long Running and MUST not have x-ms-long-running-operation property defined?